cocos/particle/particle-system.ts

```diff
@@ -769,32 +769,32 @@ export class ParticleSystem extends ModelRenderer {
     /**
      * @ignore
      */
-    private _isPlaying: boolean;
-    private _isPaused: boolean;
-    private _isStopped: boolean;
-    private _isEmitting: boolean;
-    private _needToRestart: boolean;
-    private _needRefresh: boolean;
+    private declare _isPlaying: boolean;
+    private declare _isPaused: boolean;
+    private declare _isStopped: boolean;
+    private declare _isEmitting: boolean;
+    private declare _needToRestart: boolean;
+    private declare _needRefresh: boolean;
```

Comment:
Why change them to delare?

Reply:
It is assigned in `constructor`. If don't use `declare`, there will be two assignments in the generated js file.
```js
constructor () {
    ...
    this. _isStopped = void 0;
    this. _isStopped = false;
    ...
}
```